renovate.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["config:base"],
+  "branchConcurrentLimit": 5,
+  "packageRules": [
+    {
+      "matchDatasources": ["npm"],
+      "stabilityDays": 10
```

Comment:
It looks like this field got renamed: https://docs.renovatebot.com/configuration-options/#minimumreleaseage